bpf/lib/metrics.h

```diff
@@ -56,6 +56,8 @@ static __always_inline int ct_to_metrics_dir(int ct_dir)
 		return METRIC_INGRESS;
 	case CT_EGRESS:
 		return METRIC_EGRESS;
+	case CT_SERVICE:
+		return METRIC_SERVICE;
```

Comment:
Could we reassign the `CT_XXX` values so they match our `METRICS_XXX` values 1:1 or is that already considered part of the API somehow? Doing so may help reduce complexity here. It's possible the compiler optimizes those statements enough that it doesn't matter; if the change isn't trivial, we might need to check the compiled instructions.

/cc @brb